guides/coveralls-ci-cd/coveralls_ci_cd.md

```diff
@@ -1,7 +1,9 @@
-**The following guide assumes you're using both RSpec and Cucumber for testing.**
+ **The following guide assumes you're using both RSpec and Cucumber for testing.**
+
+**The following guide assumes that you have an upstream repository on Craft Academys Github account, if that is not the case for you then just ignore the parts about adding the "Craft Academy repository" and set your own repository there instead**
```

Comment:
repetition of  "The following guide assumes"

Reply:
fixed